internal/integration_test/vs/runtime.go

```diff
@@ -183,6 +184,18 @@ func (m *wazeroModule) CallI32I32_V(ctx context.Context, funcName string, x, y u
 	return
 }
 
+func (m *wazeroModule) CallI32I32_I64(ctx context.Context, funcName string, x, y uint32) (uint64, error) {
+	results, err := m.funcs[funcName].Call(ctx, uint64(x), uint64(y))
+	if err != nil {
+		return 0, err
+	}
+	if len(results) > 0 {
```

Comment:
this should never be true, basically this is a bug if the sig is I32I32_I64, so panic or return an error.

Reply:
If I add something like

```go
	if len(results) == 0 {
		panic("unexpected")
	}
```

Then it panics, but don't know exactly why

Reply:
ok lemme pull this and take a look

Reply:
the problem is that the code was switched to the signature of "greeting" but the function name called was "greet" which didn't return a value.